tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/friendship/lpn_disable.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Copyright 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Check that disabling LPN gives correct behaviour
```

Comment:
Some additional details would be good, this is testing a very specific scenario :)